easybuild/easyconfigs/k/Keras/Keras-2.1.5-goolfc-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'Keras'
+version = '2.1.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://keras.io/'
+description = """Keras is a minimalist, highly modular neural networks library, written in Python and
+capable of running on top of either TensorFlow or Theano."""
+
+toolchain = {'name': 'goolfc', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['7ca3a381523bad40a6922e88951a316664cb088fd01cea07e5ec8ada3327e3c7']
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('Theano', '1.0.1', versionsuffix),
+    ('TensorFlow', '1.7.0', versionsuffix),
```

Comment:
@akesandgren Do you want to stick with `1.7.0`, or shall we consider bumping to `1.8.0`?